src/components/SlidePanel/SlidePanel.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import "./_slide-panel.scss";
+
+function SlidePanel({ children, onClose }) {
```

Comment:
This is just a basic pass at this panel, we'll want to enhance its functionality in follow-ups.